ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -173,6 +178,12 @@ export class WalletDB {
       ),
       valueEncoding: NULL_ENCODING,
     })
+
+    this.accountIdsToCleanup = this.db.addStore({
+      name: 'A',
```

Comment:
why we name it 'A' loll

Reply:
What would a better name be? all the stores have single letter names. The reason was because the prefix buffers weren't hashed when calculated for the store. Now they are fixed size buffers of 4 bytes to avoid collision automatically but that is 4 extra bytes per row instead of 1 byte per row.